### ISSUE TITLE
Latest version of phpstorm slowed down by this plugin

### ISSUE THREAD

**Issue by stephaniecognitif** [Konafets/antlers-idea#173]
**Describe the bug**

**OS:**
- [ X ] Mac
- [ ] Linux 
- [ ] Windows

**Jetbrains Product:**
- [X ] PhpStorm
- [ ] IntelliJ

**IDE Version:**
- [late 2023 versions to 2024.1.1]

**Plugin Version:**
- [e.g. 0.0.6]

**To Reproduce**
1. Copy paste, make comments or do anything code related in an antler file

**Expected behavior**
Smooth fast response

**Additional context**
The IDE is super slow and takes over 30 seconds just to copy paste code or comment a line

**Comment by SuperHydroman** [comment]
I actually have the same for PhpStorm 2024.1.1.

For me it's actually worse, whenever I try to edit the file, or place a comment, or copy + paste the file, the whole IDE freezes and shortly after my whole computer. I've disabled the plugin and now all the freezing and crashing is gone along with all my highlighting. Every file is completely white instead of nice syntax colors. Please fix this A.S.A.P! I work with Antlers in my job environment, this plugin is a lifesaver for our company.

**Comment by stephaniecognitif** [comment]
Yeah, my Mac does not crash completely but copy / pasting takes at least 45 seconds for me. That's if the whole IDE don't just close on me

**Comment by Konafets** [comment]
Hi,

Thanks for the report. I am currently very busy with client work but will try to look into it in the next days.

**Comment by stephaniecognitif** [comment]
I had error logs also in my IDE once, something about openapi, I'm not sure I can find them back, but if I do I'll make sure to link it here as well too.
